pydantic/validators.py

```diff
@@ -421,6 +443,9 @@ def find_validators(type_: AnyType, config: Type['BaseConfig']) -> Generator[Any
     if is_callable_type(type_):
         yield callable_validator
         return
+    if is_literal_type(type_):
+        yield make_literal_validator(getattr(type_, '__args__', getattr(type_, '__values__')))
```

Comment:
__args__ is for python 3.7, __values__ for python 3.6. Let me know if you have an alternative suggestion